examples/spaces-live-cursors/page.md

```diff
@@ -0,0 +1,122 @@
+# Live cursors for applications
```

Comment:
The title doesn't follow the same format as the Chat ones. Maybe something like "Tracking users' cursors"?